assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -67,7 +68,18 @@ function useUploader(refreshLibrary = true) {
   );
 
   const uploadFile = (file) => {
-    // TODO Add permission check here, see Gutenberg's userCan function.
+    if (!hasUploadMediaAction) {
+      const permissionError = new Error();
```

Comment:
Might be nice to create an util for creating a custom error, looks like it's used quite a few times in this file. Thoughts?

Reply:
Done